### PR TITLE
Add rake task to create request with build results

### PR DESCRIPTION
**Pull request by eduardoj**
Try it in the development environment with: `bin/rails dev:requests:request_with_build_results`

Example of running the new task in a Rails console:
```
frontend@45673968231d:/obs/src/api> bin/rails dev:requests:request_with_build_results
Creating a request which builds and produces build results...
* Request 17 has been created.
To start the builds confirm or perfom the following steps:
- Create the interconnect with openSUSE.org
- Create a couple of repositories in project home:Iggy or in project home:Iggy:branches:home:Admin
frontend@45673968231d:/obs/src/api> 
```

### CODE REVIEW

**Comment by dmarcoux** [comment]
I ran the rake task and the generated request doesn't have build results due to missing build targets for the project.

no-builds

**Comment by dmarcoux** [comment]
It's a bit confusing because the output of this new rake task does say that there will be build results, although there aren't any until we accomplish the steps you mentioned. Those steps could be in the output of the rake task.

**Comment by eduardoj** [comment]
> It's a bit confusing because the output of this new rake task does say that there will be build results for the package in the newly created request, although there aren't any until we accomplish the steps you mentioned. Those steps could be in the output of the rake task.

Agree. I added a message, after the request is created. I also updated the message in the description of the PR. Please have a look again.

**Review by saraycp — CHANGES_REQUESTED** [review]
I think we should call this task from the `dev:test_data:create` tasks defined in dev.rake.

File: src/api/lib/tasks/dev/requests.rake [hunk 1]
```diff
@@ -96,5 +96,45 @@ namespace :dev do
...
+        creator: iggy,
+        target_project: home_admin_project,
+        target_package: 'hello_world',
+        source_project: branches_iggy,
+        source_package: hello_world_iggy
+      )
+      puts "* Request #{request.number} has been created."
+      puts 'To start the builds confirm or perfom the following steps:'
+      puts '- Create the interconnect with openSUSE.org'
+      puts '- Create a couple of repositories in project home:Iggy or in project home:Iggy:branches:home:Admin'
```
Comment (saraycp) [inline src/api/lib/tasks/dev/requests.rake:137]:
```suggestion
      puts '- Create a couple of repositories in project home:Iggy:branches:home:Admin'
```

As we do the branch before we add the repos to home:Iggy, those repos are not copied to the branched project. So it doesn't build anything.

**Review by saraycp — CHANGES_REQUESTED** [review]

File: src/api/lib/tasks/dev/requests.rake (hunk 1 above)
Comment (eduardoj) [reply src/api/lib/tasks/dev/requests.rake]: Ops, true. Changed.

**Comment by eduardoj** [comment]
> I think we should call this task from the `dev:test_data:create` tasks defined in dev.rake.

Done. Please take a look again.

**Review by saraycp — APPROVED** [review]

**Pull request merged**